src/test/java/com/google/api/codegen/testdata/discoveries/csharp/csharp_dataproc.v1.json.baseline

```diff
@@ -38,108 +38,6 @@ namespace DataprocSample
                 ApplicationName = "Google Cloud Platform Sample",
             });
 
```

Comment:
I simply updated it from https://www.googleapis.com/discovery/v1/apis/dataproc/v1/rest . As to what prompted the deletion from there, I have no idea.